.github/workflows/validate-file-extensions.yaml

```diff
@@ -0,0 +1,66 @@
+name: "Validate file extensions"
```

Comment:
I´m not sure why you´re doing so many checks. I would simply verify whether any added files have the `*.yml`extension and fail if any are found.

Reply:
I tried. Now better?